src/Resources/dev/shop/payment/integrated.js

```diff
@@ -0,0 +1,180 @@
+const IntegratedPayment = {
+  options: {
+    api: null,
+    cartId: null,
+    paymentId: null,
+    paymentOptionId: null,
+    form: {},
+    checkoutForm: null,
+    token: null,
+    notValid: true,
+    fieldsValid: {
+      cardHolder: false,
+      pan: false,
+      cvv: false,
+      exp: false,
+    },
+    fieldsEmpty: {
+      cardHolder: true,
+      pan: true,
+      cvv: true,
+      exp: true,
+    },
+    inputStyle: {
+      default: {
+        color: '#2B343D',
+        fontFamily: 'Poppins, Arial, sans-serif',
+        fontSize: '14px',
+        textAlign: 'left',
+        '::placeholder': {
+          color: '#969a9f',
+        },
+        ':focus': {
+          color: '#2B343D',
+        }
+      },
+      invalid: {
+        color: '#E91932'
+      }
+    },
+    save_card: false,
+    schemes: null,
+    scheme: null,
+    query: null,
+    submit: null,
+    order_review: false,
+    return_url: null
+  },
+  init() {
+    if (payplug_integrated_payment_params === undefined) {
+      return;
+    }
+    if (payplug_integrated_payment_params.has_saved_cards) {
+      document.querySelectorAll('.payment-choice__input').forEach((element) => {
+        element.addEventListener('change', (e) => {
+          if ('payplug_choice_card_other' === e.currentTarget.id && e.currentTarget.checked) {
+            IntegratedPayment.openFields();
+            return;
+          }
+          IntegratedPayment.closeFields();
+        })
+      })
+      return;
+    }
+    if (document.querySelector('[id*=sylius_checkout_select_payment_payments][value=payplug]:checked')) {
+      IntegratedPayment.openFields();
+    }
+    document.querySelectorAll('[id*=sylius_checkout_select_payment_payments]').forEach((element) => {
+      element.addEventListener('change', (e) => {
+        if ('payplug' === e.currentTarget.value && e.currentTarget.checked) {
+          IntegratedPayment.openFields();
+          return;
+        }
+        IntegratedPayment.closeFields();
+      })
+    });
+  },
+  openFields() {
+    document.querySelector('.payplugIntegratedPayment').classList.add('payplugIntegratedPayment--loaded');
+    document.querySelector('#next-step').classList.add('disabled');
+    if (null === IntegratedPayment.options.api) {
+      IntegratedPayment.load();
+    }
+  },
+  closeFields() {
+    document.querySelector('.payplugIntegratedPayment').classList.remove('payplugIntegratedPayment--loaded');
+    document.querySelector('#next-step').classList.remove('disabled');
+  },
+  load() {
+    IntegratedPayment.options.api = integratedPaymentApi = new Payplug.IntegratedPayment(true);
```

Comment:
I think we miss to add the is_live in the `payplug_integrated_payment_params` to pass it here.